packages/openactive-integration-tests/test/features/payment/prepayment-required-unavailable/not-implemented/prepayment-required-unavailable-not-in-use-test.js

```diff
@@ -1,6 +1,6 @@
 const { FeatureHelper } = require('../../../../helpers/feature-helper');
 
-FeatureHelper.describeFeatureShouldBeImplementedIfOtherFeaturesAre(module, {
+FeatureHelper.describeFeatureShouldBeImplementedIfOtherFeaturesAreOrAreNot(module, {
```

Comment:
I find this method name really confusing but don't really have any suggestions for an alternative. Could this be split into two methods? 
`describeFeatureShouldBeImplementedIfOtherFeaturesAre`
`describeFeatureShouldBeImplementedIfOtherFeaturesAreNot`

I can't see any examples that use both (but I understand that it could in theory).

Reply:
Yes you're totally right. I've done that and that makes it easier to understand. And even if there were examples that used both, they can easily just be two separate tests